src/qt/clientmodel.cpp

```diff
@@ -127,6 +127,8 @@ void ClientModel::updateTimer()
 {
     // no locking required at this point
     // the following calls will acquire the required lock
+    Q_EMIT ChainLockHashChanged(m_node.llmq().GetBestChainLockHash());
+    Q_EMIT ChainLockHeightChanged(m_node.llmq().GetBestChainLockHeight());
```

Comment:
Should really use `NotifyChainLock` core signal instead of using the update timer here imo (similar to the way we use `MasternodeListChanged` signal for example).

Reply:
Do you have any tips on this? I'm lost